store/storetest/preference_store.go

```diff
@@ -353,15 +353,15 @@ func testPreferenceDeleteCategory(t *testing.T, ss store.Store) {
 
 	store.Must(ss.Preference().Save(&model.Preferences{preference1, preference2}))
 
-	if prefs := store.Must(ss.Preference().GetAll(userId)).(model.Preferences); len([]model.Preference(prefs)) != 2 {
+	if prefs, err := ss.Preference().GetAll(userId); len(prefs) != 2 {
```

Comment:
same here and in the rest of the `store.Must` cases of this file for the `GetAll` call.